lib/localize.js

```diff
@@ -214,6 +214,7 @@ export const Localize = class extends getLocalizeClass() {
 	constructor(config) {
 		super();
 		super.constructor.setLocalizeMarkup(localizeMarkup);
+		this.localize = (...args) => super.localize(...args);
```

Comment:
Haha really that was it?! 💯 Is there a way to have a test that would have failed before when they were sharing resources but now passes?